db_sql.go

```diff
@@ -133,17 +150,39 @@ func (x *sqlAuthenticationDB) ResetPasswordFinish(identity string, token string,
 	}
 }
 
-func (x *sqlAuthenticationDB) CreateIdentity(identity, password string) error {
+func (x *sqlUserStoreDB) CreateIdentity(identity, password string) error {
 	hash, ehash := computeAuthausHash(password)
 	if ehash != nil {
 		return ehash
 	}
 
+	//insert into user store
```

Comment:
For textual comments, leave a space after `//`
There are some other modifications which have the same spacing issue. You can find all such places with the following regex: `//[^\s]`